token-validation-spring/src/test/java/no/nav/security/token/support/spring/MultiIssuerConfigurationPropertiesTest.java

```diff
@@ -46,6 +43,12 @@ public void test() {
             "sub",
             "aud"
         );
+
+        assertTrue(config.getIssuer().containsKey("number5"));
+        assertEquals("http://metadata5", config.getIssuer().get("number5").getDiscoveryUrl().toString());
+        System.out.println(config.getIssuer().toString());
+        assertEquals(10L, config.getIssuer().get("number5").getJwkSetCache().getLifespan());
```

Comment:
Law of Demeter. These long call chains are hard to read an suggest that the distribution of responsibilities between classes is not optimal